src/chains.json

```diff
@@ -20,6 +20,7 @@
       "networkId": 1,
       "nativeCurrency": {"name":"Ether","symbol":"ETH","decimals":18},
       "web3": ["https://mainnet.infura.io/v3/${INFURA_ID}"],
+      "dappnode": "http://geth.dappnode:8545",
```

Comment:
better use fullnode.dappnode

Reply:
If you agree, I think that the best option is to have web3.infura and web3.fullnode.dappnode since all of those are actualy web3 providers.

Reply:
not sure I get what you mean - don't we need a JSON RPC endpoint and not a web3 provider?

Reply:
@ligi yes sure, it's just the naming. @Jakic007 leave it then fullnode.dappnode.